android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectDetailsFragment.java

```diff
@@ -361,10 +361,9 @@ protected Boolean doInBackground(Void... params) {
 				return false;
 			}
 			// load slideshow images
-//			slideshowImages = status.getSlideshowForProject(project.master_url);
-			if(slideshowImages == null || slideshowImages.size() == 0) return false;
-			return true;
-		}
+            // slideshowImages = status.getSlideshowForProject(project.master_url);
+            return slideshowImages != null && slideshowImages.size() != 0;
```

Comment:
When `return`ing the result of a non-trivial logic expression like this, I think you should put the expression into parentheses. It just reduces confusion to those fellow coders who are not yet used to this style of handling conditionals.

I'll refer to this comment in other changes below when I think they could use *parentheses* as well.

Reply:
Fixed.

Reply:
Fixed.